cmd/system/node.go

```diff
@@ -0,0 +1,138 @@
+package system
+
+import (
+	"fmt"
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	cp "github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func getLatestVersion(version, channel string) (*string, error) {
+	resp, err := http.Get(fmt.Sprintf("https://nodejs.org/download/%s/%s", channel, version))
+	if err != nil {
+		return nil, err
+	}
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	regex := regexp.MustCompile(`(?m)node-(.*)-linux-.*"`)
+	result := regex.FindStringSubmatch(string(body))
+	if len(result) < 2 {
+		return nil, fmt.Errorf("could not find latest version for %s", version)
+	}
+	return &result[1], nil
+}
+
+func MakeInstallNode() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "node",
+		Short: "Install Node.js",
+		Long:  `Install the Node.js JavaScript runtime ✨🐢🚀✨`,
+		Example: `arkade system install node
+  arkade system install node --version v17.9.0`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringP("version", "v", "latest", "The version for Node.js to install, either a specific version, 'latest' or ' latest-codename'")
```

Comment:
What does `latest-codename` mean?

Reply:
this are the codename of the different versions:
![image](https://user-images.githubusercontent.com/38325136/162690417-6fed27e6-fcd0-4b42-8868-be76700aba90.png)

I thought, people may want to use them instead of a specific version or latest

What do youz think @alexellis?

Reply:
latest-codename sounds like a value that you'd enter in, perhaps we could write:

`'latest' or a specific version`

Reply:
Can you add a specific version into this list as an example?

Reply:
Rather than "latest-codename", put `latest-CODENAME` so that it's obvious what you mean. I'd never heard of "codename" and have been installing Node.js for a number of years now.

Reply:
done

`The version for Node.js to install, either a specific version, 'latest' or 'latest-CODENAME' (eg: latest-gallium)`